db/schema.rb

```diff
@@ -47,6 +46,13 @@
     t.index ["reset_password_token"], name: "index_asset_managers_on_reset_password_token", unique: true
   end
 
+  create_table "building_assignments", id: false, force: :cascade do |t|
```

Comment:
naming  conventions for join tables are usually <table1 name>_<table2 name>_join_table for clarity, unless the join table means something concretely (i. e. doctors have many patients through **appointments** and vice versa). If there has to be join tables between other types of users and and buildings, `Building_assignments` might be too vague.

Reply:
This was primarily to match the name of the `BuildingAssignment` model I created to connect buildings and building operators.

I can call it `buildings_building_operators_join_table` and set the `self.table_name` to this in `BuildingAssignment`if you want me to go through with this, though. 

Or do you think renaming the `BuildingAssignment` model to `BuildingOperatorAssignment` would be better?

Reply:
BuildingOperatorsAssignment might be better :) when our code gets handed off it might make more sense to whoever is responsible for the schema